tools/rir_corpus_generator/README.md

```diff
@@ -0,0 +1,85 @@
+# RIR Corpus Generator
+
+## Outline
+
+This tool can be used to generate a corpus of room impulse responses (RIR) for a set of randomly generated rooms and microphone array placements. The tool is using [Pyroomacoustics](https://github.com/LCAV/pyroomacoustics) to simulate RIRs.
+
+The generator loads configuration from a config file, generates RIRs, saves the metadata in a manifest file and RIRs in a HDF5 file.
+
+## Features
+
+The generator is easily reconfigurable and supports configuration of the following options
+
+- Number of rooms for each subset, range of dimensions and RT60 for randomization
+- Microphone array placement and orientation randomization
+- Number of sources per room and their placement inside the room
+
+## Parameters
+
+An example of a RIR corpus generator setup is provided in `conf/rir_corpus_v1.yaml`
+
+## Example
+
+The RIR corpus generator with the example config can be used by running:
+
+```bash
+python rir_corpus_generator.py output_dir=OUTPUT_DIR
+```
+
+where `OUTPUT_DIR` is a path to the output directory.
+
+The output will be structured as
+
+```bash
+OUTPUT_DIR
++--{train, dev, test}
+|	+--*.h5
```

Comment:
Hmm, why H5? Its yet another dependency the user will need. Any other data format that is inbuilt to python / current dependencies ? Or are the outputs so large that they need h5. Note that HDFS has some issues with large number of small files, and with random indexing over large number of files (I think it cant do random access or it was much slower than sequential read last I checked)